libs/common/src/lib/upload/upload-data/upload-data.component.html

```diff
@@ -28,4 +28,13 @@ <h1>{{ 'stash.upload_data' | translate }}</h1>
       </div>
     </div>
   </div>
+  <nsi-info-card>
+    {{ 'stash.upload.go-to-synchronize' | translate }}
+    <pa-button
+      aspect="basic"
+      size="small"
+      (click)="goToSynchronize()">
```

Comment:
you can directly use `routerLink` on `pa-button` to prevent creating a function doing just a navigate:

```routerLink="../sync"
```